x/delayedack/keeper/rollapp_packet.go

```diff
@@ -25,13 +27,62 @@ func (k Keeper) SetRollappPacket(ctx sdk.Context, rollappPacket types.RollappPac
 	), b)
 }
 
-// UpdateRollappPacketStatus deletes the current rollapp packet and creates a new one with and updated status under a new key.
-// It assumes that the packet has been previously stored with the pending status.
-func (k Keeper) UpdateRollappPacketStatus(ctx sdk.Context, rollappPacket types.RollappPacket, newStatus types.RollappPacket_Status) types.RollappPacket {
+// GetRollappPacket retrieves a rollapp packet from the KVStore.
+func (k Keeper) GetRollappPacket(ctx sdk.Context, rollappPacketKey string) *types.RollappPacket {
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix(types.RollappPacketKeyPrefix))
+	b := store.Get([]byte(rollappPacketKey))
+	if b == nil {
+		return nil
+	}
+
+	var rollappPacket types.RollappPacket
+	k.cdc.MustUnmarshal(b, &rollappPacket)
```

Comment:
Don't call Must functions inside this logic
it can panic without recover

Reply:
not sure why. 
I see in a lot of places in the sdk they are callinng `mustMarshal` etc from keepers. 
I believe cosmos has a panic handler which recovers.
Are you convinced in this?
This is one example from the sdk but I saw many: https://github.com/cosmos/cosmos-sdk/blob/a35e5c3e1e7d19f149bea02778dc3861bb1b19fc/x/auth/keeper/account.go#L25